src/Provider/FreeGeoIp/FreeGeoIp.php

```diff
@@ -33,13 +34,20 @@
     private $baseUrl;
 
     /**
+     * @var string
+     */
+    private $locale;
+
+    /**
      * @param HttpClient $client
+     * @param string     $locale
      * @param string     $baseUrl
      */
-    public function __construct(HttpClient $client, string $baseUrl = 'https://freegeoip.net/json/%s')
+    public function __construct(HttpClient $client, string $locale = 'en', string $baseUrl = 'https://freegeoip.net/json/%s')
```

Comment:
I think it's better, then it will be string or null. Another way - set it with empty string, and check for if ($this->locale) in getRequest method. How do you think?